docs/pages/router/advanced/nesting-navigators.mdx

```diff
@@ -4,6 +4,14 @@ description: Learn how to nest navigators in Expo Router.
 ---
 
 import { FileTree } from '~/ui/components/FileTree';
+import { VideoBoxLink } from '~/ui/components/VideoBoxLink';
+
+<VideoBoxLink
+  videoId="izZv6a99Roo"
+  title="Using a Stack Navigator with Expo Router"
+  description="Navigate between screens, pass params between screens, create dynamic routes, and configure the screen titles and animations."
+  className="mb-6"
+/>
 
 > **warning** Navigation UI elements (Link, Tabs, Stack) may move out of the Expo Router library in the future.
```

Comment:
```suggestion
> **warning** Navigation UI elements (Link, Tabs, Stack) may move out of the Expo Router library in the future.

<VideoBoxLink
  videoId="izZv6a99Roo"
  title="Using a Stack Navigator with Expo Router"
  description="Navigate between screens, pass params between screens, create dynamic routes, and configure the screen titles and animations."
  className="mb-6"
/>
```

Let's keep the warning as the first element on this page. This will ensure that it is not something that can be missed when landing on this page.